<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectConfig</class>
 <widget class="QWidget" name="ProjectConfig">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>479</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Julia interpreter:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="JuliaInterpreter">
       <property name="placeholderText">
        <string>Full path to julia interpreter, leave empty for default</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelGraphicsDir">
       <property name="text">
        <string>Graphics file directory:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="GraphicsFileDir">
       <property name="placeholderText">
        <string>Directory for saving graphics files, e.g., /tmp/graphics</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelSocketHost">
       <property name="text">
        <string>Socket host:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="SocketHost">
       <property name="placeholderText">
        <string>Host for TCP socket, e.g., localhost</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelSocketPort">
       <property name="text">
        <string>Socket port:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="SocketPort">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>65535</number>
       </property>
       <property name="value">
        <number>8080</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelGraphicsMethod">
       <property name="text">
        <string>Graphics redirection method:</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QComboBox" name="GraphicsMethod">
       <item>
        <property name="text">
         <string>File-based</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>TCP socket</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
